pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
Even the docs use "server.crt":
https://coreos.com/etcd/docs/latest/op-guide/security.html
https://coreos.com/os/docs/latest/generate-self-signed-certificates.html